src/api/authentication/authentication_router.js

```diff
@@ -0,0 +1,111 @@
+const { response, request } = require("express");
+const router = require('express').Router();
+const bcrypt = require("bcryptjs");
+const jwt = require('jsonwebtoken');
+
+const { createFileReadStream } = require("../../core/utils/file_streams/file_read_stream");
+const { responseHandlers } = require("../../core/utils/response_handlers");
+const authenticationService = require('../../core/service/authentication-service');
+const checkAuthentication = require('../middlewares/checkAuthentication');
+const apiConfig = require('../../config/apiConfig.json');
+
+/**
+ * Fetch the Login Form
+ */
+router.get('/get-login-form', (request, response) => {
+  const filePath = __dirname + '/../../pages/login-form.html';
+
+  const readStream = createFileReadStream(filePath)
+    .on('error', (err) => {
+      console.log(err);
+      responseHandlers.internalServer(response, err);
+    });
+
+  response.setHeader('Content-Type', 'text/html');
+  readStream.pipe(response);
+});
+
+router.get('/get-register-form', (request, response) => {
+  const filePath = __dirname + '/../../pages/register-form.html';
+
+  const readStream = createFileReadStream(filePath)
+    .on('error', (err) => {
+      console.log(err);
+      responseHandlers.internalServer(response, err);
+    });
+
+  response.setHeader('Content-Type', 'text/html');
+  readStream.pipe(response);
+});
+
+router.post('/login', (request, response) => {
+  const { email, password } = request.body;
+  authenticationService.getUser(email, password)
+    .then((user) => {
+      if (user) {
+        response.status(200);
+        const jwt = createJWT(user);
+        const cookie = createCookie('access_token', jwt);
+
+        response.status(200).json({
+          redirectUrl: apiConfig.loginRegisterRedirectionURL,

```

Comment:
Why do you return this redirectUrl when you have onSuccess and onError Callbacks in the SDK when loggin in?
Is this meant to be a default case if no callbacks are passed in the SDK?

Reply:
Because the default callback function, which will be executed after successful login/register will get the redirection url from the response from back-end.